crates/rustc_codegen_spirv/src/codegen_cx/mod.rs

```diff
@@ -126,6 +127,14 @@ impl<'tcx> CodegenCx<'tcx> {
             }
         }
 
+        let mut exclusive_bindless = false;
+        for &feature in &tcx.sess.target_features {
+            if feature == sym.exclusive_bindless {
+                exclusive_bindless = true;
+                break;
+            }
+        }
```

Comment:
Probably good to merge these for loops. TBH I might prefer an enum here, and do error checking around things like an invalid combination.